.openshift-ci/sensor_versions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Returns the latest patches of the last 4 major versions
+"""
+
+import sys
+import re
+import subprocess
+
+def isReleaseTagGitCLI(version):
+    return bool(re.search(r"^\d+\.\d+\.\d+$", version))
```

Comment:
Not sure about `bool()` cast, a check for `None` seems more readable. No strong opinion.
```suggestion
    return re.search(r"^\d+\.\d+\.\d+$", version) is not None
```